sycl/test-e2e/Matrix/element_wise_abc.cpp

```diff
@@ -10,5 +10,7 @@
 // RUN: %{build} -o %t.out
 // RUN: %{run} %t.out
 
+// XFAIL: gpu-intel-dg2
```

Comment:
Adding XFAIL will make the code fail silently for the other supported combination.
It may be better to comment out the new combination until the support you are adding to IGC make it to the driver.
I don't have a strong opinion here.

Reply:
For sure! I guess we will leave this issue as draft until the IGC PR (https://github.com/intel-innersource/drivers.gpu.unified/pull/169727/files) has been merged?